### ISSUE TITLE
Feature request: exclude an arbitrary package from parse_r_script

### ISSUE THREAD

**Issue by jsta** [MangoTheCat/functionMap#45]
Not sure if this is possible. As it stands you can only exclude functions coming from base R.

**Comment by gaborcsardi** [comment]
Makes sense. On the other hand I am working on the rewrite of this package, because it is too slow. So I am not sure if it will be implemented here.
